src/renderer/missionWindow/MissionWindow.tsx

```diff
@@ -155,9 +193,18 @@ export default class MissionWindow extends Component<ThemeProps, State> {
     this.stopMissions = this.stopMissions.bind(this);
     this.toggleMissionType = this.toggleMissionType.bind(this);
     this.tipFormatter = this.tipFormatter.bind(this);
+
+    this.onChange = this.onChange.bind(this);
+    this.updateVehicles = this.updateVehicles.bind(this);
```

Comment:
why did you move line 148 to here

Reply:
![image](https://user-images.githubusercontent.com/13097797/75594541-e766cb80-5a3d-11ea-9904-fbec24bf6af2.png)